src/xo-app/new-vm/index.js

```diff
@@ -1039,48 +1040,25 @@ export default class NewVm extends BaseComponent {
 // INTERFACES ------------------------------------------------------------------
 
   _renderInterfaces = () => {
-    const { formatMessage } = this.props.intl
     const {
       state: { VIFs },
       pool
     } = this.state
     return <Section icon='new-vm-interfaces' title='newVmInterfacesPanel' done={this._isInterfacesDone()}>
       <SectionContent column>
         {map(VIFs, (vif, index) => <div key={index}>
-          <LineItem>
-            <Item label='newVmMacLabel'>
-              <DebounceInput
-                className='form-control'
-                debounceTimeout={DEBOUNCE_TIMEOUT}
-                onChange={this._getOnChange('VIFs', index, 'mac')}
-                placeholder={formatMessage(messages.newVmMacPlaceholder)}
-                rows={7}
-                value={vif.mac}
-              />
-            </Item>
-            <Item label='newVmNetworkLabel'>
-              <span className={styles.inlineSelect}>
-                {pool ? <SelectNetwork
-                  onChange={this._getOnChange('VIFs', index, 'network', 'id')}
-                  predicate={this._getNetworkPredicate()}
-                  value={vif.network}
-                />
-                : <SelectResourceSetsNetwork
-                  onChange={this._getOnChange('VIFs', index, 'network', 'id')}
-                  resourceSet={this.state.resourceSet}
-                  value={vif.network}
-                />}
-              </span>
-            </Item>
-            <Item>
-              <Button onClick={() => this._removeInterface(index)} bsStyle='secondary'>
-                <Icon icon='new-vm-remove' />
-              </Button>
-            </Item>
-          </LineItem>
+          <Interface
```

Comment:
If not too big, re-inline it (because the goal to cache the handlers is already achieved because of `linkState`).